core/src/main/java/com/amplifyframework/core/model/query/QuerySortBy.java

```diff
@@ -32,6 +34,7 @@
  * {@link QueryField#descending()} helper methods (e.g. Todo.DESCRIPTION.ascending() or Todo.DESCRIPTION.descending())
  */
 public final class QuerySortBy {
+    private final String model;
```

Comment:
```suggestion
    private final String modelName;
```
This is the identifier usually used for this concept elsewhere in the code base 

Same update suggested for all uses